src/controls/widgets/string/WidgetString.tsx

```diff
@@ -34,7 +34,8 @@ export interface Widget_string extends Widget_string_types {}
 export class Widget_string implements IWidget<Widget_string_types> {
     readonly id: string
     readonly type: 'str' = 'str'
-    get hasBlock() { return this.config.textarea ?? false } // prettier-ignore
+    // get hasBlock() { return this.config.textarea ?? false } // prettier-ignore
+    get hasBlock() { return false } // prettier-ignore
```

Comment:
Not sure how we want to handle textarea, but the collapse is broken, so this removes it.